frontend/src/components/reflection/subjectArea/subjectArea.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+
+import {
+  IonList,
+  IonText,
+  IonItem,
+  IonLabel,
+  IonBadge,
+  IonIcon,
+  IonButton,
+} from '@ionic/react'
+
+import {
+  badgeStatus,
+  checkmarkIcon,
+  currentSelectedTopic,
+  doneStatusText,
+  textLable,
+} from './subjectAreaUtils'
+
+interface TopicItem {
+  id: string
+  lable: string
+  describe: string
+  subListItems: SubTopicItem[]
+}
+
+export interface SubTopicItem {
+  id: string
+  lable: string
+  done: boolean
+}
+
+export type InputSelectValue = string | 'Später Wählen'
+
+interface Props {
+  selectValue: (input: InputSelectValue) => void
+  topicList: TopicItem[]
+}
+
+/*
+    The Component is designed to not store the selected value in the second level
+    You must Provide a Callback-Function where the value is emitted to and store
+    the state outside of this Component
+ */
+export const SubjectArea: React.FC<Props> = ({ topicList, selectValue }) => {
```

Comment:
```suggestion
export const ReflexionSubjectArea: React.FC<Props> = ({ topicList, selectValue }) => {
```
that way we no it belongs to `Reflexions`

Reply:
done